### DISCUSSION TITLE
When the onDragging event is fired?

### DISCUSSION THREAD

**Discussion by kolligs-nwp** [bvaughn/react-resizable-panels#278] [Q&A] [answered]
When I add a function to the ResizeHandle component's onDragging callback, it never gets called. Am I using it wrong?

`<ResizableHandle onDragging={(dragging) => {
            dragging ? console.log('Dragging') : console.log('dragging stopped');
          }} />`

**Comment by bvaughn** [comment]
This looks like a bug.  I'm going to move this discussion to an issue, #279

**Comment by bvaughn** [comment, marked as answer]
Fixed in v2.0.3 (just published)

---

❤️ → ☕ [givebrian.coffee](http://givebrian.coffee/)
